client/driver/rkt.go

```diff
@@ -0,0 +1,201 @@
+                "trust",
+                "--prefix=" + trust_prefix,
+        }
+        // Add the given trust prefix
+        var outBuf, errBuf bytes.Buffer
+        cmd := exec.Command(trust_args[0], trust_args[1:]...)
+        cmd.Stdout = &outBuf
+        cmd.Stderr = &errBuf
+        d.logger.Printf("[DEBUG] Starting rkt command: %q", strings.Join(trust_args, " "))
+        if err := cmd.Start(); err != nil {
+                return nil, fmt.Errorf(
+                        "Error running rkt: %s\n\nOutput: %s\n\nError: %s",
+                        err, outBuf.String(), errBuf.String())
+        }
+        d.logger.Printf("[DEBUG] Added trust prefix: %q", trust_prefix)
+
+        name, ok := task.Config["name"]
+        if !ok || name == "" {
+                return nil, fmt.Errorf("Missing ACI name for rkt")
+        }
+        run_args := []string{
+                "rkt",
+                "run", "--interactive", name,
+        }
+        // Run the ACI
+        var aoutBuf, aerrBuf bytes.Buffer
+        acmd := exec.Command(run_args[0], run_args[1:]...)
+        acmd.Stdout = &aoutBuf
+        acmd.Stderr = &aerrBuf
+        d.logger.Printf("[DEBUG] Starting rkt command: %q", strings.Join(run_args, " "))
+        if err := acmd.Start(); err != nil {
+                return nil, fmt.Errorf(
+                        "Error running rkt: %s\n\nOutput: %s\n\nError: %s",
+                        err, aoutBuf.String(), aerrBuf.String())
+        }
+        d.logger.Printf("[DEBUG] Started ACI: %q", name)
+        h := &rktHandle{
+                proc:   acmd.Process,
+                name:   name,
+                doneCh: make(chan struct{}),
+                waitCh: make(chan error, 1),
+        }
+        go h.run()
+        return h, nil
+}
+
+func (d *RktDriver) Open(ctx *ExecContext, handleID string) (DriverHandle, error) {
+        // Parse the handle
+        pidBytes := []byte(strings.TrimPrefix(handleID, "Rkt:"))
+        qpid := &rktPID{}
+        if err := json.Unmarshal(pidBytes, qpid); err != nil {
+                return nil, fmt.Errorf("failed to parse Rkt handle '%s': %v", handleID, err)
+        }
+
+        // Find the process
+        proc, err := os.FindProcess(qpid.Pid)
+        if proc == nil || err != nil {
+                return nil, fmt.Errorf("failed to find Rkt PID %d: %v", qpid.Pid, err)
+        }
+
+        // Return a driver handle
+        h := &rktHandle{
+                proc:   proc,
+                name:   qpid.Name,
+                doneCh: make(chan struct{}),
+                waitCh: make(chan error, 1),
+        }
+
+        go h.run()
+        return h, nil
+}
+
+func (h *rktHandle) ID() string {
+        // Return a handle to the PID
+        pid := &rktPID{
+                Pid:  h.proc.Pid,
+                Name: h.name,
+        }
+        data, err := json.Marshal(pid)
+        if err != nil {
+                log.Printf("[ERR] failed to marshal rkt PID to JSON: %s", err)
```

Comment:
You will want to pass the logger from the Driver into the Handle. We support redirecting logs to files or other sources so this would break that it interface by using the default logger.